README.md

```diff
@@ -26,6 +26,8 @@ Or install it yourself as:
 ## Usage
 
 ```ruby
+SemanticRange.valid?('1.2.3') # true
+SemanticRange.valid?('a.b.c') # false
```

Comment:
Could add a test for `valid?(123)` to make sure it catches the first guard clause, as well as a test for a strippable string > `MAX_LENGTH` characters.

Reply:
Good idea, added those to the spec